### ISSUE TITLE
kraken.js -  irrecoverable error: General:Invalid arguments:volume

### ISSUE THREAD

**Issue by gvdhoven** [askmike/gekko#1735]
**Note: for support questions, please join our [Discord server](https://discord.gg/26wMygt)**

* **I'm submitting a ...**
[X] bug report
[ ] feature request
[ ] question about the decisions made in the repository

* **Action taken** (what you did)
Started live trader via UI

* **Expected result** (what you hoped would happen)
Bot keeps trading in the background

* **Actual result** (unexpected outcome)
Trader crashed

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, etc)
```
2018-01-20 10:55:28 (DEBUG):    [kraken.js] entering "setBalance" callback after kraken-api call, data: { error: [],
  result:
   { ZEUR: '0.0000',
     XXBT: '0.0005241800',
     XXRP: '0.00000000',
     XETH: '0.0000049300',
     XREP: '0.0000000000',
     XICN: '0.0000063300',
     XMLN: '0.0000000000',
     EOS: '298.8050716400',
     BCH: '0.0000079200' } }
2018-01-20 10:55:28 (INFO):     Attempting to BUY 0.5013677666188426 EOS at kraken price: 0.00104080
2018-01-20 10:55:28 (DEBUG):    [kraken.js] (addOrder) BUY 0.5013677 EOS @ 0.0010408 XBT
2018-01-20 10:55:29 (ERROR):    [kraken.js] (addOrder) returned an irrecoverable error: General:Invalid arguments:volume

```

**Comment by fouman75** [comment]
You might want to look at exchanges/kraken.js around line 330. You can change the minimum amount for your currency pair.

Check out this Kraken page: https://support.kraken.com/hc/en-us/articles/205893708-What-is-the-minimum-order-size-

You might want to try with min amount 0.02 for XBT/EOS based on the above site.

You might also look at this site to adjust the precision.
https://blog.kraken.com/post/1261/announcement-reducing-price-precision-on-all-trading-pairs/)

I'm just starting with Gekko but this seemed to fix the error for me on another pair. Still testing it out though...

**Comment by gvdhoven** [comment]
@fouman75 according to their page, the XBT/EOS need to be 3(!), not 0.02. As you can see i tried to buy 0.5 which is higher then 0.02 and that failed. According to Kraken site:

> The minimum order size goes by **base currency** (**base currency is the left currency in each pair - e.g. XBT is the base currency in the pair XBT/USD**)

So the file needs to be updated according to the minimums mentioned over there.

**Comment by cmroche** [comment]
We recently merged PR #1550 into the develop branch which uses a script and properly populates order amount for Kraken. Please sync the develop branch and give that a try as I suspect it will fix this issue.
